### DISCUSSION TITLE
GPG import key crash the smart card

### DISCUSSION THREAD

**Discussion by fabioluciano** [solokeys/solo2#158] [General]
I just created a gpg key, and tried to import to solo2, and the key crashed... I needed to remove from computer to start working again.

![image](https://user-images.githubusercontent.com/499057/225458266-1ea783b7-06a5-4141-b218-75c3e0d01882.png)

I even can list the device using the cli

![image](https://user-images.githubusercontent.com/499057/225458543-ac48a684-0be2-4a96-8e4e-4f2b088d2290.png)

**Comment by Andrysky** [comment]
the question is, is gpg support implemented? - mine is not .. (
and is it planned to add it?

**Comment by baldurmen** [comment]
I've hit the same issue, and solved it this way: https://github.com/solokeys/solo2-cli/issues/107#issuecomment-1595655657
